nomenclature/code.py

```diff
@@ -209,3 +210,17 @@ class RegionCode(Code):
     """
 
     hierarchy: str = None
+    countries: List[str] = None
+
+    @validator("countries")
+    def check_iso3_codes(cls, v, values) -> bool:
+        invalid_iso3_codes: List[str] = []
+        for country in v:
+            if pycountry.countries.get(alpha_3=country) is None:
+                invalid_iso3_codes.append(country)
+        if invalid_iso3_codes:
+            raise ValidationError(
+                f"Region {values['name']} has invalid\n"
+                " ISO3 country codes: {invalid_iso3_codes}"
```

Comment:
You'll need another f here to make string interpolation work
```suggestion
                f" ISO3 country codes: {invalid_iso3_codes}"
```